poppler: 20.08.0 -> 20.10.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/poppler-glib/versions.


meta.description for poppler is: "A PDF rendering library"

meta.homepage for poppler is: "https://poppler.freedesktop.org/"

meta.changelog for poppler is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 20.10.0 with grep in /nix/store/lnbbmigdmgc0wbnqz1inhpcbyzidq5ah-poppler-glib-20.10.0
- directory tree listing: https://gist.github.com/718f25c79c4292ac1a80e2f8af9f89e7

- du listing: https://gist.github.com/739e62608eac317b7cda46798908c1f3


---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
1498 total rebuild path(s)

470 package rebuild(s)

470 x86_64-linux rebuild(s)
428 i686-linux rebuild(s)
170 x86_64-darwin rebuild(s)
430 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
R
adapta-gtk-theme
aesop
apostrophe
apvlv
arc-theme
arx-libertatis
asciidoc-full
asciidoc-full-with-plugins
asymptote
auctex
autofs5
baresip
beamerpresenter
bibata-cursors
bibata-cursors-translucent
bibata-extra-cursors
bookworm
btrbk
calibre
calibre-py2
calibre-py3
calligra
capitaine-cursors
ccache
ccacheStdenv
ccacheWrapper
cddlib
chrome-gnome-shell
cinnamon.cinnamon-screensaver
cinnamon.iso-flags-png-320x420
cinnamon.iso-flags-svg
clevis
cloudcompare
cups-filters
dblatex
dblatexFull
denemo
diff-pdf
diffpdf
digikam
disorderfs
dot2tex
dropbox-cli
dwarf-fortress-packages.dwarf-fortress-full
dwarf-fortress-packages.dwarf-therapist
dwarf-fortress-packages.dwarf-therapist-original
dwarf-therapist
eaglemode
eclib
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/lnbbmigdmgc0wbnqz1inhpcbyzidq5ah-poppler-glib-20.10.0 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A poppler https://github.com/r-ryantm/nixpkgs/archive/c81f2e820209336c7be07f881336c7a46a462f97.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/lnbbmigdmgc0wbnqz1inhpcbyzidq5ah-poppler-glib-20.10.0
ls -la /nix/store/lnbbmigdmgc0wbnqz1inhpcbyzidq5ah-poppler-glib-20.10.0/bin
```

---

</details>
<br/>



### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @ttuegel @jtojnar @worldofpeace for testing.

jtojnar commented:
Announcements look okay:
- 20.09.0: https://lists.freedesktop.org/archives/poppler/2020-September/014432.html
- 20.10.0: https://lists.freedesktop.org/archives/poppler/2020-October/014467.html
- 20.11.0: https://lists.freedesktop.org/archives/poppler/2020-November/014499.html

New version has been released in the meanwhile.

jtojnar commented:
@GrahamcOfBorg build cups-filters texlive.bin scribusUnstable inkscape

jtojnar committed:
poppler: 20.08.0 -> 20.12.1

- 20.09.0: https://lists.freedesktop.org/archives/poppler/2020-September/014432.html
- 20.10.0: https://lists.freedesktop.org/archives/poppler/2020-October/014467.html
- 20.11.0: https://lists.freedesktop.org/archives/poppler/2020-November/014499.html
- 20.12.0: https://lists.freedesktop.org/archives/poppler/2020-December/014546.html
- 20.12.1: https://lists.freedesktop.org/archives/poppler/2020-December/014562.html

jtojnar commented:
@GrahamcOfBorg build cups-filters texlive.bin scribusUnstable inkscape

jtojnar commented:
@GrahamcOfBorg build cups-filters texlive.bin scribusUnstable inkscape

jtojnar commented:
New announcements:
* 20.12.0 https://lists.freedesktop.org/archives/poppler/2020-December/014546.html
* 20.12.1 https://lists.freedesktop.org/archives/poppler/2020-December/014562.html
